packages/office-ui-fabric-react/src/components/ActivityItem/ActivityItem.tsx

```diff
@@ -9,6 +9,11 @@ import { getStyles } from './ActivityItem.styles';
 import { PersonaSize, PersonaCoin, IPersonaSharedProps } from '../../Persona';
 
 export class ActivityItem extends BaseComponent<IActivityItemProps, {}> {
+  public static defaultProps: Partial<IActivityItemProps> = {
+    beaconColorOne: '#00FFEC',
```

Comment:
do we need themable colors for this or is it just a shine?

Reply:
verify this looks okay in a dark theme

Reply:
I got the colors from Coachmark - confirming with design what to do here.

Reply:
We switched to comparable theme colors. @phkuo how do you suggest testing dark theme? Is there a SP palette we can plug-in to the demo site?